src/wp-content/themes/twentytwentyfour/functions.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Twenty Twenty Four functions and definitions
+ *
+ * @link https://developer.wordpress.org/themes/basics/theme-functions/
+ *
+ * @package Twenty Twenty Four
+ * @since Twenty Twenty Four 1.0
+ */
+
+/**
+ * Register block Styles
+ */
+
+if ( ! function_exists( 'twentytwentyfour_block_styles' ) ) :
+	/**
+	 * Register custom block styles
+	 *
+	 * @return void
+	 * @since Twenty Twenty-Four 1.0
+	 *
+	 */
+	function twentytwentyfour_block_styles() {
+		/**
+		 * The wp_enqueue_block_style() function allows us to enqueue a stylesheet
+		 * for a specific block. These will only get loaded when the block is rendered
+		 * (both in the editor and on the front end), improving performance
+		 * and reducing the amount of data requested by visitors.
+		 *
+		 * See https://make.wordpress.org/core/2021/12/15/using-multiple-stylesheets-per-block/ for more info.
+		 */
+		wp_enqueue_block_style(
+			'core/button',
+			array(
+				'handle' => 'twentytwentyfour-button-style-outline',
+				'src'    => get_template_directory_uri() . '/assets/css/button-outline.css',
+			)
```

Comment:
```suggestion
						'ver'    => wp_get_theme()->get( 'Version' ),
			)
```

By default, if a version (`ver`) is not specified in the array, then it defaults to `false`. When version is empty for a registered script or stylesheet, then it defaults to using the current version of WordPress being run.

Let's change this to pass the theme version so that caches are specific to the version of the theme being used and not Core. The latter could result in no cache busting when the theme is updated on its own without updating WordPress after.

The same suggestion above should be applied to any script or style registration.

Reply:
makes sense, thank you!